formulas/wick-base/functions/wick-make-user

```diff
@@ -92,9 +96,14 @@ wickMakeUser() {
             args[${#args[@]}]=$setShell
         fi
 
-        if [[ -n "$setGroups" ]]; then
+	if [[ -n "$groups" ]]; then
+	    wickStringSplit groupsAsArray "$groups"
+            for group in "${groupsAsArray[@]}"; do
```

Comment:
Please add a newline above.

A lot of this is covered in the style guide, cited in the `CONTRIBUTING.md` file.